airflow/providers/fab/provider.yaml

```diff
@@ -26,7 +26,7 @@ description: |
 
 # The provider is not yet ready to be released, we will skip it by default when preparing new release waves
 # For providers until we think it should be released.
-state: not-ready
```

Comment:
I'm just wondering could we create Release Candidates for `not-ready` states, upload it to pypi but do not release final version?

E.g. with this timeline:
Wave Provider on YYYY-MM-DD: apache-airflow-providers-fab -> rc1
Wave Provider on YYYY-MM-DD + 2 week : apache-airflow-providers-fab -> rc2
Wave Provider on YYYY-MM-DD + 4 week : apache-airflow-providers-fab -> rc3
State: ready
Wave Provider on YYYY-MM-DD + 6 week : apache-airflow-providers-fab -> rc4 -> release

Or this would increase complexability of the release process?

Reply:
Well. There are quite a few considerations there, because of the way how`pip` treats pre-release packages in a different namespace and how `chicken-egg` providers works. Unfortunatley `--include-pre` is not selective and it includes all pre-release packages when we are installing them in CI

Reply:
Because in CI we are relasing and testing `.dev0` packages, and `>=2.9.0` is a different namespace thant 2.9.0.dev0, so the `pypi` packages would have to have `>=2.9.0.dev0`. Also we need to release the final `fab` package anyway before we release 2.9.0, so things get a little complicated when it comes to number of edge-cases to handle.

Reply:
Yeah if need need to resolve a lot of cases then it's not worth spend time to this. Especially that it is not common when new provider required non-released Airflow version.